content/reports/240315-contributors.md

```diff
@@ -0,0 +1,33 @@
+---
+type: contributors
+description: |
+  Hive version for TDP stack 2.0
```

Comment:
As we discuss, can you use the terms `Hadoop 3.3 stack` instead of `TDP stack 2.0`

Reply:
I have already used `TDP stack 2.0` in the last report, so changing the name in each report will be very confusing. Until we set the official name, I suggest we keep the name TDP stack 2.0. However, I can say in the report that it is not the definitive name and that the stack uses `Hadoop 3.3`.